src/plugins/intel_cpu/tests/functional/subgraph_tests/src/reshape_inplace.cpp

```diff
@@ -81,5 +84,59 @@ class InPlaceReshapeFromConstantCheck : public SubgraphBaseTest {
 TEST_F(InPlaceReshapeFromConstantCheck, smoke_CPU_InPlaceReshapeFromConstantCheck) {
     run();
 }
+
+/* Case 2:
+ * Subgraph
+ *
+ *         params[0]   params[1]
+ *                \     /
+ *                 \   /
+ *                  add---reshape2---result2
+ *                   |
+ *                reshape1
+ *                   |
+ *                  MVN
+ *                   |
+ *                result1
+ *
+ *  If parent of reshape is shared, reshape in place should be not applicable.
+ *  This is becuase multiple branches could change data on the same port, then pollute result each other.
```

Comment:
```suggestion
 *  The same memory is shared between the `result2` input and `MVN` output. The CPU graph inplace memory conflict
 *  resolution logic must prevent `result2` data being rewritten by the MVN node.
```
In fact, a Reorder node is inserted to prevent data corruption within the in place conflict resolution pass.

Reply:
Thank you for the explanation. I actually dumped execution graph for the first version subgraph test case(one reshape inside), and found there is a reorder inserted, so not reproduce, but guess the reason is memory alignment or sth others, now it's clear.
Updated. Thanks Maksim!